Annotation/constituent_golden_set.json

```diff
@@ -402,7 +421,55 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how they plan on incorporating the strategic plan",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "discuss",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",
+                "clause": "who will be involved",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "discuss",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",
+                "clause": "how detailed of a plan they need for their organization",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "discuss",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",
+                "clause": "how they plan on communicating it",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "discuss",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "constituent",

```

Comment:
I would remove "if they have a board" from the clause

Reply:
I think you have to argue that the "if" clause is also embedded tho, otherwise you cannot get the structure of the sentence